core/src/test/scala/unit/kafka/security/authorizer/AclAuthorizerTest.scala

```diff
@@ -988,6 +980,30 @@ class AclAuthorizerTest extends ZooKeeperTestHarness {
     }
   }
 
+  @Test
+  def testAuthorizeByResourceTypeNoAclFoundOverride(): Unit = {
+    testAuthorizeByResourceTypeNoAclFoundOverride(aclAuthorizer)
+  }
+
+  private def testAuthorizeByResourceTypeNoAclFoundOverride(authorizer: Authorizer): Unit = {
```

Comment:
The `authorizer` parameter is not used. Can't we just move this into the test method `testAuthorizeByResourceTypeNoAclFoundOverride` above?

Reply:
Yes. commit b6a766b